alluxiofs/core.py

```diff
@@ -24,13 +24,16 @@ def __init__(
         logger=None,
         concurrency=64,
         http_port="28080",
+        preload_path=None,
         **kwargs,
     ):
         super().__init__(**kwargs)
         self.logger = logger or logging.getLogger("AlluxioFileSystem")
         self.alluxio = AlluxioSystem(
             etcd_host, worker_hosts, options, logger, concurrency, http_port
         )
+        if preload_path is not None:
```

Comment:
Is it possible to 
```
alluxio = fsspec.filesystem("s3", worker_hosts="localhost")
alluxio.submit_load("s3://siyuan-overmind-bucket/overmind/2/Security3xPddm")
alluxio.progress.....
```
Not quire sure about how python interface work, whether we are able to call alluxio specific methods

Reply:
I think This should not work.